spellings/tools/find-unknown-comment-words

```diff
@@ -82,12 +82,21 @@ then
 fi
 
 TMPFILE=${0##*/}-$USER-$RANDOM
+
 unknowns=( "not-used" )     # get around empty array with nounset
 # Symlinks will be ignored in this spell check. `-type f` switch in `find` command will ignore symlinks.
-extract-comments `find $DIRNAME \( -iname \*.[ch] -o -iname \*.dox \) -type f` |
+
+# Ignore errors here.
```

Comment:
What is the purpose of ignore errors for the `extract-comments` command?

Reply:
Removed! :)
I was seeing some issues with the error switched on. So, I had disabled it. But now they seem to be gone. I tested on FreeRTOS and HTTP. See [this PR](https://github.com/FreeRTOS/coreHTTP/pull/103).

Fingers crossed that this doesn't cause an issue with other repos. If it does, we'll follow up with a quick PR.